tests/integ/conftest.py

```diff
@@ -63,26 +79,71 @@ def nonce(request):
     return request.config.getoption("--nonce")
 
 
-@pytest.yield_fixture(autouse=True)
-def cleanup_objects(engine):
-    yield
-
-    # TODO track bound models w/model_bound signal (TODO), then use boto3 to scan/delete by Meta.table_name
-    # Running tests individually may break if the User table isn't bound as part of that test
-    users = list(engine.scan(User))
-    engine.delete(*users)
-
-
 @pytest.fixture(scope="session")
-def dynamodb():
-    return fixed_ddb_client
+def dynamodb(do_provide_aws):
+    return fixed_session.client("dynamodb", endpoint_url=resource_options['endpoint_url'])
 
 
 @pytest.fixture(scope="session")
-def dynamodbstreams():
-    return fixed_streams_client
+def dynamodbstreams(do_provide_aws):
+    return fixed_session.client("dynamodbstreams", endpoint_url=resource_options['endpoint_url'])
 
 
 @pytest.fixture
 def engine(dynamodb, dynamodbstreams):
-    return Engine(dynamodb=dynamodb, dynamodbstreams=dynamodbstreams)
+    yield Engine(dynamodb=dynamodb, dynamodbstreams=dynamodbstreams)
+
+
+def get_open_port():
+    with contextlib.closing(
```

Comment:
Please add a comment explaining what this does.  It looks like it reserves a port, then releases it (with `.closing`) and returns the value immediately.  Is this the best way to query for an available port?

Reply:
done